db/migrate/20230123192359_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] When you check the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram,you notice that in the `Post` table, we have `AuthorID`. It therefore means that there's an `Author` table, that links to the `User` table as an alias. I therefore think that your reference should be:
- `:post, :author, foreign_key: { to_table: 'users' }`

Reply:
well resolved! ✔️